app/javascript/packages/address-search/components/post-office-no-results.tsx

```diff
@@ -0,0 +1,23 @@
+import { getAssetPath } from '@18f/identity-assets';
+import { t } from '@18f/identity-i18n';
+
+function PostOfficeNoResults({ address }) {
+  return (
+    <>
+      <img
+        className="grid-col-2 inline-block veritcal-align-top margin-top-22"
+        style={{ display: 'inline-block', marginTop: '20px', verticalAlign: 'top' }}
+        alt="exclamation mark inside of map pin"
```

Comment:
we should probably add the alt text to the translation files and also get translations for Spanish and French.

Reply:
Good idea. I will reach out to design about getting translations.

Reply:
sp: Imagen de un marcador de mapa

fr: Image d'une épingle à carte

Reply:
Image of a map pin

Reply:
@svalexander  Translations are now applied.